notebooks/00_quick_start/dkn_single_node.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DKN : Deep Knowledge-Aware Network for News Recommendation\n",
+    "DKN\\[1\\] is a deep learning model which incorporates information from knowledge graph for better news recommendation. Specifically, DKN uses TransX\\[2\\] method for knowledge graph representaion learning, then applies a CNN framework, named KCNN, to combine entity embedding with word embedding and generate a final embedding vector for a news article. CTR prediction is made via an attention-based neural scorer. \n",
+    "\n",
+    "## Properties of DKN:\n",
+    "- DKN is a content-based deep model for CTR prediction rather than traditional ID-based collaborative filtering. \n",
+    "- Make use of knowledge entities and common sense in news content via joint learning from semantic-level and knnowledge-level representations of news articles.\n",
+    "- DKN uses an attention module to dynamically calculate a user's aggregated historical representaition.\n",
+    "\n",
+    "## Data format:\n",
+    "one simple example:\n",
+    "`0 CandidateNews:1050,5238,5165,6996,1922,7058,7263,0,0,0 entity:0,0,0,0,2562,2562,2562,0,0,0 clickedNews0:7680,3978,1451,409,4681,0,0,0,0,0 entity0:0,0,0,395,0,0,0,0,0,0 clickedNews1:3698,2301,1055,6481,6242,7203,0,0,0,0 entity1:0,0,1904,1904,1904,0,0,0,0,0 ...`\n",
```

Comment:
I don't understand this very well:
`one simple example: 0 CandidateNews:1050,5238,5165,6996,1922,7058,7263,0,0,0 entity:0,0,0,0,2562,2562,2562,0,0,0 clickedNews0:7680,3978,1451,409,4681,0,0,0,0,0 entity0:0,0,0,395,0,0,0,0,0,0 clickedNews1:3698,2301,1055,6481,6242,7203,0,0,0,0 entity1:0,0,1904,1904,1904,0,0,0,0,0`
In the notebook, all the lines appear in the same line, would you mind to separate them in different lines?

Reply:
Sure. Done.